project/RELEASE-DETAILS.md

```diff
@@ -0,0 +1,43 @@
+# Release Details
+
+This documents the process for tagging the VIC repository for a release and publishing the VIC binaries on bintray.com
+
+## Tag The Release
+
+* Follow the [RELEASE-VERSIONING.md](RELEASE-VERSIONING.md) for choosing the new version.
+* Gather the Release Notes file.
+* Go to https://github.com/vmware/vic/releases and click on Draft a new release.
+* Add the tag version that meets the [RELEASE-VERSIONING.md](RELEASE-VERSIONING.md) requirements for this release.
+* Title the release "vSphere Integrated Containers Version X.Y.Z" where X.Y.Z meets the versioning requirements. 
+* Paste the release note file contents into the Write field and preview.
+* If this is release is non-production ready select the "This is a pre-release" box.
+
+
+## Create the VIC binary package
```

Comment:
Should add a TODO here and/or link to an issue, the upload from drone to bintray can all be automated via drone.yml.
The only manual part should be creating the git tag - we need to add the git tag steps to this doc in any case.